src/test/java/core/basesyntax/service/FruitTransactionParserTest.java

```diff
@@ -0,0 +1,113 @@
+package core.basesyntax.service;
+
+import static org.hamcrest.CoreMatchers.is;
+
+import core.basesyntax.model.Fruit;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.service.impl.FruitTransactionParserImpl;
+import java.util.ArrayList;
+import java.util.List;
+import org.junit.Assert;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class FruitTransactionParserTest {
+    private static final String INFO_LINE = "type,fruit,quantity";
+    private static final String FIRST_TRANSACTION_LINE = "b,apple,50";
+    private static final String SECOND_TRANSACTION_LINE = "p,apple,40";
+    private static final String FIRST_INCORRECT_TRANSACTION_LINE = "p.orange;100";
+    private static final String SECOND_INCORRECT_TRANSACTION_LINE = "";
+    private static final String THIRD_INCORRECT_TRANSACTION_LINE = null;
+    private static List<String> firstIncorrectData;
+    private static List<String> secondIncorrectData;
+    private static List<String> thirdIncorrectData;
+    private static FruitTransaction firstTransaction;
+    private static FruitTransaction secondTransaction;
```

Comment:
Use it as local variables in your tests. It's difficult to read and understand your tests with variables like this